Templater/Daily Note.md

```diff
@@ -1,25 +1,31 @@
 ---
+<%*const file = tp.file.find_tfile(moment(tp.file.title,"YYYY-MM-DD").subtract(1,'days').format("YYYY-MM-DD")); const fileCache = app.metadataCache.getFileCache(file); let Totals_Task_Done = 1; if (fileCache?.frontmatter["Totals_Task_Done"]) { Totals_Task_Done = fileCache.frontmatter["Totals_Task_Done"]; }-%>
+
 creation Date: <% tp.file.creation_date() %>
 tags:
   - Daily
   - <%tp.file.title%>
-  - <% moment(tp.file.title, "YYYY-MM-DD").format("YYYY-[W]ww") %>
-
+  - <%moment(tp.file.title,"YYYY-MM-DD").format("YYYY-[W]ww")%>
 linklist:
-  - "[[<%tp.date.now("YYYY-[W]ww",0)%>]]"
-aliases:
-Totals_Task-Done: 
-Totals_Task-ToDo:
-Totals_Task-total:
+  - '[[<% moment(tp.file.title, "YYYY-MM-DD").format("YYYY-[W]ww") %>]]'
+aliases: 
+Totals_Task_Done: <%Totals_Task_Done%>
+Totals_Task_ToDo: 
+Totals_Task_totals: 
 Daily_New_task:
 ---
 
-Days: [[Periodic Notes/1.Daily Notes/<%tp.date.now("YYYY-MM-DD",-1,tp.file.title,"YYYY-MM-DD")%>|Yesterday]] <-- **[[<%tp.date.now("YYYY-MM-DD",0,tp.file.title,"YYYY-MM-DD")%>]]** --> [[Periodic Notes/1.Daily Notes/<%tp.date.now("YYYY-MM-DD",1,tp.file.title,"YYYY-MM-DD")%>|Tomorrow]]
```

Comment:
Doucment how this works and how you would recreate it using momend.js in the documentation files